### PR TITLE
Msw 100522400 mailer nil parameter

### PR DESCRIPTION
**Pull request by michaelwehrley**
Update Params of Mailer

* Add default parameter to nil.
* Remove logger statements.

[Finished: #100522400]

### CODE REVIEW

File: app/mailers/comment_mailer.rb [hunk 1]
```diff
@@ -1,10 +1,10 @@
 # CommentMailer is used to notify recipients of comments via email.
 class CommentMailer < ActionMailer::Base
-  def comment_email_alert(receiving_participant, message_body, *subject)
+  def comment_email_alert(receiving_participant, message_body, subject = nil)
     logger.info("INFO BEFORE: Comment email notification sent \
```
Comment (ericcf) [inline app/mailers/comment_mailer.rb]: Is there a good reason to keep the logger statement?

**Comment by ericcf** [comment]
One of your commits is titled "Increase Spec Coverage", but it doesn't look like any specs were updated/added. Am I missing something?

**Comment by michaelwehrley** [comment]
The spec coverage was added in a previous commit, but I didn't realize the Eric merged the branch until I committed this one.  So yes, the description and branch name aren't quite right.  As for the logger statements, those were in there and so I didn't want to delete them if we didn't have a good reason to.  I am not sure how else we would see if emails weren't sent unless we Inform sentry?

**Comment by ericcf** [comment]
Ok, maybe it's worth an interactive rebase to name and/or squash the commits?

Those logger statements have nothing to do with Sentry. They get written to the log and probably ignored! Here's the library we're using for Sentry: https://github.com/getsentry/raven-ruby

**Comment by michaelwehrley** [comment]
Yep, they have nothing to do with sentry. I guess I meant "log" statements instead of emails.  My question was that instead of logger statements, should we use Sentry.  And I take it from your comment, that is what you would recommend.  And as for the commit, I can update it.

**Comment by ericcf** [comment]
No, I don't think Sentry makes sense for these because they're just debugging statements. It would be better to notify us of failures.

**Comment by ericcf** [comment]
I have no further objections!

**Comment by michaelwehrley** [comment]
lol.  Thanks!

**Pull request merged**